frontend/src/model/beta/Releases.tsx

```diff
@@ -6,21 +6,21 @@ import { ModelInterface } from '../../../types/v2/types'
 import { sortByReleaseVersionDescending } from '../../../utils/arrayUtils'
 import EmptyBlob from '../../common/EmptyBlob'
 import Loading from '../../common/Loading'
-import DraftNewReleaseDialog from '../DraftNewReleaseDialog'
-import ModelReleaseDisplay from '../ModelReleaseDisplay'
+import DraftNewReleaseDialog from './releases/DraftNewReleaseDialog'
+import ModelReleaseDisplay from './releases/ModelReleaseDisplay'
 
 export default function Releases({ model }: { model: ModelInterface }) {
   const [latestRelease, setLatestRelease] = useState<string>('')
   const [openDraftNewRelease, setOpenDraftNewRelease] = useState(false)
 
-  const { releases, isReleasesLoading } = useGetReleasesForModelId(model.id)
+  const { releases, isReleasesLoading, mutateReleases } = useGetReleasesForModelId(model.id)
 
   const sortedReleases = useMemo(() => releases.sort(sortByReleaseVersionDescending), [releases])
 
   const modelReleaseDisplays = useMemo(
     () =>
       sortedReleases.reduce<ReactElement[]>((releaseDisplays, release) => {
-        if (release.name) {
+        if (release.semver) {
```

Comment:
Releases must always have a semver, so this if statement never applies.  This `reduce` statement should be replaced with a `map`.